test/e2e/recordNode.ts

```diff
@@ -0,0 +1,44 @@
+import { spawnSync } from "child_process";
+import fs from "fs";
+
+import config from "./config";
+import { tmpFile } from "./utils";
+
+function getRecordingId(file: string) {
+  try {
+    const contents = fs.readFileSync(file).toString().split("\n")[0];
+    if (contents.length) {
+      // Ignore any trailing URL.
+      const spaceIndex = contents.indexOf(" ");
+      if (spaceIndex != -1) {
+        return contents.substring(0, spaceIndex);
+      }
+      return contents;
+    }
+    return null;
+  } catch (e) {
+    return null;
+  }
+}
+
+export async function recordNodeExample(scriptPath: string) {
+  if (!config.nodePath) {
+    console.log(`Skipping test: RECORD_REPLAY_NODE not set`);
+    return;
+  }
+
+  const recordingIdFile = tmpFile();
```

Comment:
This can totally be a follow-up, but for simplicity I would love to remove all of this random tempfile stuff, and instead use a predictable scheme based on the test-name and current time. I like to have values that have at least some clear meaning, makes debugging stuff a lot easier!